src/Perl6/World.nqp

```diff
@@ -4890,6 +4909,7 @@ class Perl6::World is HLL::World {
     # Checks if a given symbol is declared.
     method is_name(@name) {
         my int $is_name := 0;
+        self."!no_experimental_rakuast"(@name);
```

Comment:
Wouldn't this cause a serious compile time degradation?  Checking for RakuAST *every* time a name is checked ??